src/steamship_langchain/file_loaders/text.py

```diff
@@ -0,0 +1,30 @@
+"""Import text files to Steamship workspace."""
+from typing import Any, Dict, List, Optional
+
+from steamship import Block, File, MimeTypes
+
+from steamship_langchain.file_loaders import BaseFileLoader, generate_file_tags
+
+
+class TextFileLoader(BaseFileLoader):
+    """Load a text file into a Steamship workspace.
+
+    Creates a new `File` by uploading the content of local file. File tags
+    for identifying the source file and the time of import are automatically added,
+    as well as for any custom metadata that is provided. This enables query-based
+    retrieval for downstream processing.
+    """
+
+    def load(self, path: str, metadata: Optional[Dict[str, Any]] = None) -> List[File]:
+        """Load from file path."""
+        tags = generate_file_tags(client=self.client, file_path=path, metadata=metadata)
```

Comment:
Fine to punt this until later, but `generate_file_tags` and `generate_url_tags` could likely be promoted into main python SDK

Reply:
agreed. I think these importers could ultimately live there as well.